hwilib/commands.py

```diff
@@ -226,9 +231,11 @@ def process_commands(args):
     getkeypool_parser.set_defaults(func=getkeypool)
 
     displayaddr_parser = subparsers.add_parser('displayaddress', help='Display an address')
-    displayaddr_parser.add_argument('path', help='The BIP 32 derivation path of the key embedded in the address')
-    displayaddr_parser.add_argument('--sh_wpkh', action='store_true', help='Display the p2sh-nested segwit address associated with this key path')
-    displayaddr_parser.add_argument('--wpkh', action='store_true', help='Display the bech32 version of the address associated with this key path')
+    displayaddr_parser.add_argument('--desc', help='Pseudo-descriptor for which to return a descriptor with keys, e.g. wpkh(00000000/84h/1h/0h/0/*). Alternatively, use --path with --sh_wpkh / --wpkh')
+    displayaddr_parser.add_argument('--path', help='Derivation path, default follows BIP43 convention, e.g. m/84h/0h/0h/1/*')
```

Comment:
You can use a [mutually exclusive group](https://docs.python.org/3/library/argparse.html#mutual-exclusion) here that is also required so that argparse can handle errors in argument format for us.